package.json

```diff
@@ -4,7 +4,11 @@
   "description": "Webpack Encore is a simpler way to integrate Webpack into your application",
   "main": "index.js",
   "scripts": {
-    "test": "mocha --reporter spec test --recursive",
+    "test": "run-s test:*",
+    "test:normal": "mocha --reporter spec test --recursive --ignore test/persistent-cache/*",
+    "test:persistent-cache-clear": "cross-env CLEAR_PERSISTENT_CACHE=true npm run test:persistent-cache:base",
+    "test:persistent-cache": "npm run test:persistent-cache:base",
+    "test:persistent-cache:base": "mocha --reporter spec test/persistent-cache --recursive",
```

Comment:
isn't this one also run directly by the `test` script ?

Reply:
It isn't, that would require the [globstar `**`](https://github.com/mysticatea/npm-run-all/blob/HEAD/docs/run-s.md#glob-like-pattern-matching-for-script-names) instead of the used `*`